app/views/includes/help_details.html

```diff
@@ -1,10 +1,20 @@
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
 {% block help_info %}
-    <details class="help-panel help-details">
-        <summary><span class="summary help-details-summary">{{common.helpTitle}}</span></summary>
-        <div class="panel panel-border-narrow">
-            <p>{{common.helpText | safe}}</p>
-            <p>{{common.contactTelLabel}}: {{helpline.number}} ({{helpline.hours}})</p>
-            <p>{{common.helpEmailLabel}}: <a href="mailto:{{helpline.email}}">{{helpline.email}}</a></p>
-        </div>
-    </details>
+    {% set helpContent = "<p class=\"govuk-heading-s\">" + (common.helpHeading1) + "</p>" %}
+    {% set helpContent = helpContent + "<p class=\"govuk-body\">" + globals.helpline.number + "</p>" %}
+    {% set helpContent = helpContent + "<p class=\"govuk-body\">" + globals.helpline.hours + "</p>" %}
+    {% set helpContent = helpContent + "<p class=\"govuk-heading-s\">" + (common.helpHeading2) + "</p>" %}
+    {% set helpContent = helpContent + "<p class=\"govuk-body\">" + (common.helpEmailLabel | replace("{contactEmailAddress}", globals.helpline.email)) + "</p>" %}
+
+    {% if fields.featureToggles.value.webchat == "true" %}
+        {% set helpContent = helpContent + "<p class=\"govuk-heading-s\" id=\"ctsc-web-chat-header\">" + (common.helpHeading3) + "</p>" %}
```

Comment:
Yes, I can't have it as an <h_> tag as this is a common subtemplate and I don't know in advance what other <h_> tags appear before it, so just styling a <p> tag as a small heading... won't hurt anyone :)

Reply:
😕

Reply:
Say you have this as an <h3>.... on a page that has <h1> and <h2> it would be fine, but if you have another page with only an <h1> it would would throw a WCAG validation error (if I remember correctly) as it would be missing the <h2>.

Reply:
Don't have to convince me. perhaps wait for your accessibility testing and convince them. 

But i think you may have an issue here as technically its discriminatory. You show one thing for people who can see and another for people who cant.

Reply:
Why? Not following... :-/

Reply:
well you show what appears to be a heading (i.e. emphasised text) for people who can see and what appears to be a paragraph for people who cant

Reply:
its the reason we have h1,h2... tags and p tags....

Reply:
People who can't see will have a screen reader that will just read the content of the tag, right?

Reply:
yes, they also mention if its a title or not, emphasised or not etc etc. however i dont know enough about it. just that your page should be correctly formatted and syntactically correct. im sure when you get accessibility tested they will have more of an argument to satisfy you.

Reply:
I don't know enough about accessibility stuff either tbh, but I can see that Axe-Coconut doesn't raise it as an error... 😐